blocks/block.js

```diff
@@ -360,6 +373,23 @@ class Block extends Component {
       params
     }
   }
+
+  static getIndices (component) {
```

Comment:
I'm not sure if it's a good idea to have two `getIndices` methods; I think one should be purged. It was originally a method for Block because it's fairly Block-specific, but then I believe the static one is being used for Stacks and Inputs. I'm not sure if this should be moved to Component since it doesn't apply to all components though